Core/GDCore/IDE/Events/EventsPropertyReplacer.cpp

```diff
@@ -69,16 +71,21 @@ class GD_CORE_API ExpressionPropertyReplacer
   void OnVisitNumberNode(NumberNode& node) override {}
   void OnVisitTextNode(TextNode& node) override {}
   void OnVisitVariableNode(VariableNode& node) override {
+    if (isParentTypeAVariable) {
+        // Do nothing, it's a variable.
+        if (node.child) node.child->Visit(*this);
```

Comment:
In other words, we must continue to explore because properties could be used in children (becuase a variable node can have sub variables, or bracket accessors, which allows to enter a sub-expression inside the brackets (`[MyProperty]`. `[1 + MyProperty]`...), and this sub-expression must be explored to rename the property that could be there.